pkgs/applications/networking/cluster/kpt/default.nix

```
@@ -0,0 +1,29 @@
+{ buildGoModule, fetchFromGitHub,  fetchgit, fetchhg, fetchbzr, fetchsvn, lib }:
```

`fetchgit`, `fetchhg`, `fetchbzr`, `fetchsvn` are note used

```suggestion
{ buildGoModule, fetchFromGitHub, lib }:
```

---

Done.